airbyte-workers/src/main/java/io/airbyte/workers/config/WorkerConfigurationBeanFactory.java

```diff
@@ -36,7 +37,9 @@ public class WorkerConfigurationBeanFactory {
   private static final String AIRBYTE_ROLE = "AIRBYTE_ROLE";
   private static final String AIRBYTE_VERSION = "AIRBYTE_VERSION";
   private static final String DEPLOYMENT_MODE = "DEPLOYMENT_MODE";
+  private static final String DOCKER = "DOCKER";
```

Comment:
Looks like we're mixing constants related to a few concepts, is it worth grouping in some semantic manner? (listing all the expected environments together for example)

Reply:
@gosusnp Once the dust settles, I think it is worth doing an audit of these things and moving it out to central place to avoid constant duplication/drift.